packages/core/src/errors-list.ts

```diff
@@ -169,6 +169,11 @@ export const ERRORS = {
       message:
         "The next nonce for %sender% should be %expectedNonce%, but is %pendingCount%. Please make sure not to send transactions from %sender% while running this deployment and try again.",
     },
+    BASE_FEE_EXCEEDS_GAS_LIMIT: {
```

Comment:
What do you think of adding it to the whitelist. Based on the rules I think it can be argued either way, but I suspect the stacktrace just isn't useful in this context.

Reply:
I'm on board with that idea